Makefile

```diff
@@ -12,7 +12,7 @@ endif
 
 rwildcard=$(foreach d,$(wildcard $(addsuffix *,$(1))),$(call rwildcard,$(d)/,$(2)) $(filter $(subst *,%,$(2)),$(d)))
 
-all: test build
+all: lint test build
```

Comment:
The lint target is actually not executed during the pr build, can you add that here? https://github.com/raffis/gitops-zombies/blob/b8c6400e39a8aa63cf82cb78ab65aa4f5bcce3a5/.github/workflows/e2e.yaml#L27